src/patternfly/components/TextInputGroup/text-input-group-control.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-text-input-group__control{{#if text-input-group-control--modifier}} {{text-input-group-control--modifier}}{{/if}}"
```

Comment:
Oh, I forgot to mention - is this used? Doesn't look like it, unless I missed it.